### PR TITLE
test movie-service

### PR DESCRIPTION
**Pull request by namnn17b3**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/76420

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by namnn17b3** [comment]
ready

File: test/movie.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    movieRepository = container.resolve(MovieRepository);
+  });
+
+  afterAll(async () => {
+    await AppDataSource.destroy();
+  });
+
+  it('Return new movies when input empty', async () => {
+    const newMovies = await movieService.getNewestFilms();
+    expect(JSON.stringify([11, 31, 32, 33, 34, 35])).toEqual(
```
Comment (huongnt-2545) [inline test/movie.service.spec.ts]: `[11, 31, 32, 33, 34, 35]` -> data này là gì đây e?

**Comment by huongnt-2545** [comment]
commented

**Comment by huongnt-2545** [comment]
commented

**Comment by namnn17b3** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**